src/app/search/search.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="main">
+  <section>
+    <header>
+      <h2>Do a search for...</h2>
+
+      <div class="form-group">
+        <input (change)="search(1)" [(ngModel)]="searchTerm" placeholder="search by title or description"/><!-- TODO: delay search until finish typing -->
+      </div>
+
+      <div class="form-group">
+        <span class="left">
+          <label [attr.for]="searchSeries">Filter by Series</label>
+          <select id="searchSeries" [(ngModel)]="searchSeriesId" (ngModelChange)="searchBySeries()">
+            <option *ngFor="let seriesId of allSeriesIds" [value]="seriesId">{{allSeries[seriesId] ? allSeries[seriesId].title : 'No Series'}}</option>
+          </select>
+          <!-- TODO: there should be a way to clear this -->
```

Comment:
Sigh. I miss react-select. Are there any designs on doing something like this for selects? https://plnkr.co/edit/JcG8uO9nIfSGMEKdLf0Y?p=preview A select component like this would take care of the can't clear the selection problem, and the multi select might be good for tag based searching.